### PR TITLE
Add LetsGrief.net

### PR DESCRIPTION
**Pull request by InvalidTeslaa**
## Type of change

- [X] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

## Further comments
[TeslaNetwork.zip](https://github.com/LabyMod/server-media/files/13740494/TeslaNetwork.zip)

### CODE REVIEW

**Comment by InvalidTeslaa** [comment]
[teslanetwork.zip](https://github.com/LabyMod/server-media/files/13740575/teslanetwork.zip)

**Comment by maaaathis** [comment]
- Invalid file type 'GIF454' for file: minecraft_servers/teslanetwork/icon.png
- Invalid icon size! Size is 512x512px, must be 256x256px: minecraft_servers/teslanetwork/icon.png
- Invalid file type 'GIF224' for file: minecraft_servers/teslanetwork/icon@2x.png
- Invalid hDPI icon size! Size is 256x256px, must be 512x512px: minecraft_servers/teslanetwork/icon@2x.png

**Comment by InvalidTeslaa** [comment]
icon@2x
icon

**Comment by maaaathis** [comment]
> Changed the Icon
> 
> icon@2x icon

You need to make a commit to your branch to update the icons in the pull request @InvalidTeslaa.

**Comment by InvalidTeslaa** [comment]
> > Changed the Icon
> > icon@2x icon
> 
> You need to make a commit to your branch to update the icons in the pull request @InvalidTeslaa.

How i do that? Im not very familiar with GitHub

**Comment by maaaathis** [comment]
> > > Changed the Icon
> > > icon@2x icon
> > 
> > 
> > You need to make a commit to your branch to update the icons in the pull request @InvalidTeslaa.
> 
> How i do that? Im not very familiar with GitHub

You remove the current icons here and upload the new ones: https://github.com/InvalidTeslaa/server-media/tree/master/minecraft_servers/teslanetwork
The pull request will be updated automatically.

**Comment by InvalidTeslaa** [comment]
> > > > Changed the Icon
> > > > icon@2x icon
> > > 
> > > 
> > > You need to make a commit to your branch to update the icons in the pull request @InvalidTeslaa.
> > 
> > 
> > How i do that? Im not very familiar with GitHub
> 
> You remove the current icons here and upload the new ones: https://github.com/InvalidTeslaa/server-media/tree/master/minecraft_servers/teslanetwork The pull request will be updated automatically.

Done, have reuploaded the new ones

**Comment by maaaathis** [comment]
The manifest has to be a json file.

**Comment by maaaathis** [comment]
Is your website online?
image

**Comment by InvalidTeslaa** [comment]
> Is your website online? image

Website hosting is expired and during issues with paypal im currently not able to renew it, maybe in a few days its online again

**Comment by InvalidTeslaa** [comment]
great now nothing works anymore, images and manifest are booth invalid

**Comment by maaaathis** [comment]
> great now nothing works anymore, images and manifest are booth invalid

Your manifest is not a valid json file. You need to close the "gamesmodes" object and remove the comma after "social"."discord".
Try https://jsonlint.com for validation of your manifest.

**Comment by InvalidTeslaa** [comment]
> > great now nothing works anymore, images and manifest are booth invalid
> 
> Your manifest is not a valid json file. You need to close the "gamesmodes" object and remove the comma after "social"."discord". Try https://jsonlint.com for a validation of your manifest.

Updated

**Comment by maaaathis** [comment]
> > > great now nothing works anymore, images and manifest are booth invalid
> > 
> > 
> > Your manifest is not a valid json file. You need to close the "gamesmodes" object and remove the comma after "social"."discord". Try https://jsonlint.com for a validation of your manifest.
> 
> Updated

Thats my corrected version **with your gamesmodes**, feel free to update.
```
{
    "server_name": "teslanetwork",
    "nice_name": "TeslaNetwork",
    "direct_ip": "mc.teslaa.net",
    "supported_languages": [
        "en",
        "de"
    ],
    "social": {
        "web": "https://teslaa.net",
        "web_support": "https://teslaa.net/support",
        "discord": "https://discord.gg/teslaa"
    },
    "gamemodes": {
        "citybuild": {
            "name": "CityBuild",
            "color": "#FFA500",
            "versions": "1.17<*"
        },
        "survival": {
            "name": "Survival",
            "color": "#ADD8E6",
            "versions": "1.17<*"
        }
    }
}
```

**Comment by InvalidTeslaa** [comment]
Ah i see, i forgot to add a "}", changed

> > > > great now nothing works anymore, images and manifest are booth invalid
> > > 
> > > 
> > > Your manifest is not a valid json file. You need to close the "gamesmodes" object and remove the comma after "social"."discord". Try https://jsonlint.com for a validation of your manifest.
> > 
> > 
> > Updated
> 
> Thats my corrected version **with your gamesmodes**, feel free to update.
> 
> ```
> {
>     "server_name": "teslanetwork",
>     "nice_name": "TeslaNetwork",
>     "direct_ip": "mc.teslaa.net",
>     "supported_languages": [
>         "en",
>         "de"
>     ],
>     "social": {
>         "web": "https://teslaa.net",
>         "web_support": "https://teslaa.net/support",
>         "discord": "https://discord.gg/teslaa"
>     },
>     "gamemodes": {
>         "citybuild": {
>             "name": "CityBuild",
>             "color": "#FFA500",
>             "versions": "1.17<*"
>         },
>         "survival": {
>             "name": "Survival",
>             "color": "#ADD8E6",
>             "versions": "1.17<*"
>         }
>     }
> }
> ```

**Comment by toohard2explain** [comment]
The logo is copyrighted and not yours, i don't think you're allowed to use that image.

**Comment by InvalidTeslaa** [comment]
> The logo is copyrighted and not yours, i don't think you're allowed to use that image.
I just made a screenshot of an gif from tenor 🤔

**Comment by toohard2explain** [comment]
> > > The logo is copyrighted and not yours, i don't think you're allowed to use that image.
> > I just made a screenshot of an gif from tenor 🤔

I dont think that this makes it better tbh... you should make or buy your own serverlogo (maybe on fiverr)

**Comment by InvalidTeslaa** [comment]
What about this one? 🤔

**Comment by toohard2explain** [comment]
> What about this one? 🤔

you can't just take tesla's logo and recolor it, its a brandmark that is protected. you have to make something completely new when you want to be legal

**Comment by maaaathis** [comment]
> great now nothing works anymore, images and manifest are booth invalid

I don't know the exact reason why the image validation pipeline fails, but try compressing your backgroundfiles (https://tinypng.com/) and make sure they are the right size: https://github.com/LabyMod/server-media/blob/master/docs/Files.md

**Comment by InvalidTeslaa** [comment]
> > What about this one? 🤔
> 
> you can't just take tesla's logo and recolor it, its a brandmark that is protected. you have to make something completely new when you want to be legal

Now i make a new one, this will work until we have a new one which will be permanently

**Comment by toohard2explain** [comment]
> > > What about this one? 🤔
> > 
> > 
> > you can't just take tesla's logo and recolor it, its a brandmark that is protected. you have to make something completely new when you want to be legal
> 
> Now it have a new one, this will work until we have a new one which will be permanently

you did not even remove the old one and the wordmark is also copyrighted, you cannot just use it, i dont even know if you're allowed to use the name at all

**Comment by InvalidTeslaa** [comment]
Which old one, they got overwritten

**Comment by toohard2explain** [comment]
> Which old one, they got overwritten

no you created `icon@2 (2)`

**Comment by InvalidTeslaa** [comment]
> > Which old one, they got overwritten
> 
> no you created `icon@2 (2)`

Oh, bad. My fault

**Comment by toohard2explain** [comment]
you still can't use the tesla wordmark, its also protected. you have to make something completely new..

**Comment by InvalidTeslaa** [comment]
So we changed our Servername so is it now possible to get this resolved?

**Comment by maaaathis** [comment]
> So we changed our Servername so is it now possible to get this resolved?

Compress your images and add a icon.png. See reference here: https://github.com/LabyMod/server-media/blob/master/docs/Files.md#filestructure

**Comment by InvalidTeslaa** [comment]
And yeah the server is not reachable due to cloudflare checks

https://cdn.teslaa.net/2024/01/07/88b2b3d1-4039-4793-be16-fd5b7204eec5.png

**Comment by InvalidTeslaa** [comment]
All is done

File: minecraft_servers/letsgrief/background.png
Comment (DoJapHD) [inline minecraft_servers/letsgrief/background.png:1]: Compress image

File: minecraft_servers/letsgrief/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/letsgrief/background@2x.png:1]: Compress image

File: minecraft_servers/letsgrief/manifest.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "server_name": "letsgrief",
+    "nice_name": "LetsGrief",
+    "direct_ip": "mc.letsgrief.net",
+    "supported_languages": [
+        "en",
+        "de"
+    ],
+    "social": {
+        "web": "https://letsgrief.net",
```
Comment (DoJapHD) [inline minecraft_servers/letsgrief/manifest.json:10]: Website not reachable; remove if non-existent

File: minecraft_servers/letsgrief/manifest.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "server_name": "letsgrief",
+    "nice_name": "LetsGrief",
+    "direct_ip": "mc.letsgrief.net",
+    "supported_languages": [
+        "en",
+        "de"
+    ],
+    "social": {
+        "web": "https://letsgrief.net",
+        "web_support": "https://letsgrief.net/support",
```
Comment (DoJapHD) [inline minecraft_servers/letsgrief/manifest.json:11]: Website not reachable; remove if non-existent

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Server in general not reachable; Reopen once reachable again

**Comment by InvalidTeslaa** [comment]
Domain now working, Server reachable same as Website

**Comment by InvalidTeslaa** [comment]
Unable to reopen request so i have to make a new one

**Pull request closed without merging**